content/en/docs/application-store/built-in-apps/memcached-app.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Memcached App"
+keywords: 'Kubernetes, KubeSphere, Memcached, app-store'
+description: 'How to use built-in Memcached Object Storage'
+
+
+weight: 2242
+---
+Memcached is designed for large data caches. Its API is available for most popular languages. This guide will show you one-click deployment for Memcached in Kubenetes .
```

Comment:
Add an external link with Memcached website here.

Reply:
Got it